src/components/SocialSection.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Divider from '../images/img_divider.png';
+import { getSubjectLocation, getThumbnailSource } from '../lib/get-subject-location';
 
-const SocialSection = ({ classifications, completedSubjects, percentComplete, project, subjects, volunteers }) =>
+const SocialSection = ({ classifications, completedSubjects, percentComplete, project, subjects, subjectsOfNote, volunteers }) =>
   <div className="home-page__social-section">
-    <h2 className="main-title">Subjects<i>of</i>  Note</h2>
-    <img role="presentation" className="divider" src={Divider} />
-    <div className="home-page__subject-row flex-row">
-      <div></div>
-      <div></div>
-      <div></div>
-      <div></div>
-    </div>
+
+    {subjectsOfNote.length && (
```

Comment:
Will `apiClient.type('subjects').get(newestSubjects)` ever return a `null` ? If so, this should be `{subjectsOfNote && subjectsOfNote.length && {`